bundle/CMakeLists.txt

```diff
@@ -121,27 +121,14 @@ endif()
      ecbuild_bundle( PROJECT fv3-jedi-data SOURCE "${JCSDA_DATA_ROOT}/fv3-jedi-data" )
    endif()
    if(MPAS_DYCORE)
-     set(DESTINATION_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
-     file( COPY "${JCSDA_DATA_ROOT}/mpas-jedi-data" DESTINATION ${DESTINATION_DIR} )
+     ecbuild_bundle( PROJECT mpas-jedi-data SOURCE "${JCSDA_DATA_ROOT}/mpas-jedi-data" )
```

Comment:
@guoqing-noaa  For mpas jedi, we need to copy data from (normally) other user's account to our own accounts because in mpas-jedi's run, the app needs write-permission.

Reply:
Thanks, @TingLei-NOAA 
I did not test this part. Do you know whether the jcsda ctest writes to the directory or overwrites some files?

Also, I actually have a question on this part. You can see that in the current RDASApp, 'mpas-jedi-data' is not included by an ecbuild_bundle command, I added it.  Why we don't need ecbuild_bunld for mpas-jedi-data currently?

Reply:
Tag @SamuelDegelia-NOAA @delippi to see if they have some ideas for my questions in the above. Thanks!

Reply:
> Also, I actually have a question on this part. You can see that in the current RDASApp, 'mpas-jedi-data' is not included by an ecbuild_bundle command, I added it. Why we don't need ecbuild_bunld for mpas-jedi-data currently?
I think we are using mpas-jedi-data from a single specified location (I think from Cory's dir)  and skip that step to put it into ./sorc dir.